packages/_helpers/clickable.tsx

```diff
@@ -54,8 +55,8 @@ export function Clickable({
 
   return radio || checkbox ? (
     <ToggleItem
-      labelClassName={props.labelClassName}
-      className="focus-ring focus-ring-inset absolute inset-0 h-full w-full appearance-none cursor-pointer"
+      labelClassName={classNames(props.labelClassName)}
+      className={`focus:focusable:focus focus-visible:focusable:focus-visible not-focus-visible:focusable:focus:not(:focus-visible) focusable-inset ${ccClickable.clickable}`} //TODO should be rewritten when fixed in drive, only `focusable` should be needed
```

Comment:
This is needed to be fixed in Drive([the task](https://trello.com/c/pjBacYeN/222-fix-focusable-issue)) but I would like to leave it here as it works